apps/server/src/modules/h5p-editor/controller/api-test/h5p-editor.api.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { ExecutionContext, INestApplication } from '@nestjs/common';
+import { Test } from '@nestjs/testing';
+import { ICurrentUser } from '@src/modules/authentication';
+import { JwtAuthGuard } from '@src/modules/authentication/guard/jwt-auth.guard';
+import { Request } from 'express';
+import request from 'supertest';
+import { H5PEditorModule } from '../../h5p-editor.module';
+
+class API {
+	constructor(private app: INestApplication) {}
+
+	async get(path: string) {
+		return request(this.app.getHttpServer()).get(`/h5p-editor/${path}`);
+	}
+}
+
+describe('H5PEditor Controller (api)', () => {
+	let app: INestApplication;
+	let api: API;
+
+	let currentUser: ICurrentUser;
+
+	beforeAll(async () => {
+		const module = await Test.createTestingModule({
+			imports: [H5PEditorModule],
+		})
+			.overrideGuard(JwtAuthGuard)
+			.useValue({
+				canActivate(context: ExecutionContext) {
```

Comment:
Can you please check https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/modules/lesson/controller/api-test/lesson-delete.api.spec.ts
We want to avoid the always authentication hack in future. In the linked file is a working example without this, that use 2 Helper for it.

Reply:
Now using the same approach for authentication